package.json

```diff
@@ -23,15 +23,16 @@
     "LICENSE",
     "README.md"
   ],
+  "type": "module",
   "exports": {
     "./package.json": "./package.json",
     ".": {
       "types": "./dist/index.d.ts",
       "import": "./dist/index.modern.js",
-      "require": "./dist/index.js"
+      "require": "./dist/index.cjs"
     }
   },
-  "main": "dist/index.js",
+  "main": "dist/index.cjs",
```

Comment:
updating `main` based on the build manager, `microbundle`'s, instructions (https://github.com/developit/microbundle#-installation--setup--)